_projects/2023/100385252/100385252.Rmd

```diff
@@ -0,0 +1,660 @@
+---
+title: "Life expectancy increased in all countries of the world"
+description: "The aim of this project is to reproduce, layer by layer, a chart on Life Expectancy evolution from Our World in Data"
+author: Alejandra Costa
+cataegories: "2023"
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+Across the world, people are leaving longer. In 1800, the average global life expectancy was 32 years. Nevertheless, by 2012 this figure had more than doubled, reaching 70 years. But when and where has this improvement in life expectancy occurred?
+
+In order to understand such worldwide increase in longevity, we can look at the "Life Expectancy of the World Population in 1800, 1950 and 2012" chart, sourced from Our World in Data. Using information on population and life expectancy at birth from Gapminder.org, the mentioned chart shows how living standards have increased over the long run through the visualization of life expectancy evolution over the world.
+
+Accordingly, this post aims to replicate the provided chart, as well as to propose potential enhancements and alternatives.
+
+## 1. Getting and preparing the data
```

Comment:
Please remove the numbering from titles to keep the style of the rest of the projects.